packages/esm-home-app/src/index.ts

```diff
@@ -41,47 +41,47 @@ function setupOpenMRS() {
     extensions: [
       {
         id: 'active-visits-link',
-        slot: 'app-menu-slot',
+        slot: '',
```

Comment:
You can just delete these lines completely. I will improve the documentation to cover this.

Reply:
Hi @brandones, sure i will update the changes.